src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -33,6 +33,14 @@
 
 namespace Microsoft.IdentityModel.Tokens
 {
+    /// <summary>
+    /// Definition for AcceptedAlgorithmValidator
+    /// </summary>
+    /// <param name="algorithm">The algorithm to validate.</param>
+    /// <param name="securityKey">THe securityKey to validate.</param>
+    /// <returns><c>true</c> if the algorithm is considered valid</returns>
+    public delegate bool AcceptedAlgorithmValidator(string algorithm, SecurityKey securityKey);
```

Comment:
while not specifically required, we usually pass the SecurityToken when validating as this gives users the opportunity to apply additional logic.